Add `junit5-migration-maven-kotlin` sample #105 (#107)
junit-team/junit5-samples: junit5-migration-maven-kotlin/README.md

# junit5-migration-maven-kotlin [![Build Status](https://travis-ci.org/junit-team/junit5-samples.svg?branch=master)](https://travis-ci.org/junit-team/junit5-samples)

The junit5-migration-maven-kotlin project demonstrates how to execute tests written in Java and Kotlin
based on JUnit 5 using Maven. In addition, it showcases that existing JUnit 4 based tests can be executed

> in the same test suite as JUnit Jupiter based tests or any other tests supported on the JUnit Platform.

Reviewer: ```suggestion
within the same test suite as JUnit Jupiter tests or any other tests supported by the JUnit Platform.
```

> Please note that this project is uses the [Maven Wrapper](https://github.com/takari/maven-wrapper)

Reviewer: ```suggestion
Please note that this project uses the [Maven Wrapper](https://github.com/takari/maven-wrapper)
```

> 3.6.1 version. This helps you ensure that already tested versions are not going to be failed if

Reviewer: ```suggestion
Version 3.6.1. This ensures that previously tested versions remain functional and are not disrupted if
```

> locally installed different maven version. 

Reviewer: ```suggestion
a different Maven version installed locally
```
